vip-portal/src/main/resources/vm/wrapper.vm

```diff
@@ -19,6 +19,23 @@ function error {
   echo [ ERROR - $D ] $* >&2
 }
 
+function function_singularity {
+    cat > singularity << 'EOF'
+          #!/bin/bash
+          /cvmfs/dirac.egi.eu/container/apptainer/bin/apptainer "$@"
```

Comment:
Not a good idea to have an hardcoded path like that. We will use a environment variable for the apptainer location, like `APPTAINER_LOCATION`.

Reply:
Done. The path for the $APPTAINER_LOCATION should be declared in gaswConfiguration?